cmd/kinflate/docs/workflows.md

```diff
@@ -0,0 +1,164 @@
+[OTS]: glossary.md#off-the-shelf
+[apply]: glossary.md#apply
+[applying]: glossary.md#apply
+[base]: glossary.md#base
+[fork]: https://guides.github.com/activities/forking/
+[manifest]: glossary.md#manifest
+[off-the-shelf]: glossary.md#off-the-shelf
+[overlays]: glossary.md#overlay
+[patch]: glossary.md#patch
+[patches]: glossary.md#patch
+[rebase]: https://git-scm.com/docs/git-rebase
+[resources]: glossary.md#resources
+[workflowBespoke]: workflowBespoke.jpg
+[workflowOts]: workflowOts.jpg
+
+# workflows
+
+A _workflow_ is the sequence of steps one takes to
+use and maintain a configuration.
+
+
+## No local files
+
+One just installs some [OTS] configuration from the web,
+[applying] it directly to the cluster.
+
+This assumes a configuration specified at https://github.com/kinflate/ldap.
+
+> ```
+> kinflate inflate -f https://github.com/kinflate/ldap |\
```

Comment:
We don't support manifest off the HTTP URL yet, so we should take this out ?

Reply:
It's so easy to add though 😃